File: backend/ApiServer/Database/Seed.cs
using System;
using LinqToDB;
using Database.Models;
using Serilog;
using System.Collections.Generic;
using MongoDB.Driver;
using System.Linq;
using MongoDB.Bson;

namespace Database;

public static class Seeder
{
  static Status[] Statuses = new Status[] {
    new Status()
    {
      Id = Guid.NewGuid(),
      Title = VersionedValue<string>.From("Todo") ,
      Index = VersionedValue<int>.From(1),
      Color = VersionedValue<string>.From("text-gray-500")
    },
    new Status()
    {
      Id = Guid.NewGuid(),
      Title = VersionedValue<string>.From("In Progress") ,
      Index = VersionedValue<int>.From(2),
      Color = VersionedValue<string>.From("text-amber-500")
    },
    new Status()
    {
      Id = Guid.NewGuid(),
      Title = VersionedValue<string>.From("Done") ,
      Index = VersionedValue<int>.From(3),
      Color = VersionedValue<string>.From("text-green-600")
    }
  };

  public static void Seed(DbContext dbContext)
  {
    Log.Debug("Seeding");
    if (dbContext.Statuses.AsQueryable().Count() == 0)
    {
      dbContext.Statuses.InsertMany(Statuses);
    }
  }
}

File: frontend/src/worker/seed.ts
import { insert, insertMany } from 'blinkdb';
import { v4 as uuid } from 'uuid';
import { Database } from './db';
import { Project } from './models/project';
import { Task } from './models/task';

export async function seed(db: Database) {
	const project: Project = {
		id: 'aa27f680-d644-454c-bad4-1db11c0d762c',
		title: 'My Project',
		statuses: [
			{
				id: 'ba7c2ac3-276a-458d-8d4d-ab0f3b5a7e05',
				title: 'Todo',
				order: 1,
				color: 'text-gray-500',
			},
			{
				id: '94331c09-30d0-4c7b-8748-4ba9b023a2b9',
				title: 'In Progress',
				order: 2,
				color: 'text-amber-500',
			},
			{
				id: '3641dc00-90df-4c8f-b56c-b49d17addbd9',
				title: 'Done',
				order: 3,
				color: 'text-green-600',
			},
		],
	};

	const tasks = new Array(10000).fill(null).map(
		(_, ind): Task => ({
			id: uuid(),
			projectId: project.id,
			statusId: project.statuses[0].id,
			description: [{ type: 'p', children: [{ text: '' }] }],
			title: 'Some Title',
			slug: 'MP-' + (ind + 1),
			index: ind,
		}),
	);

	await insert(db.project_table, project);
	await insertMany(db.task_table, tasks);
}
